cosmo/monitors/dark_monitors.py

```diff
@@ -41,13 +44,14 @@ def dark_filter(df_row, filter_pha, location):
                         'ROOTNAME': df_row['ROOTNAME']})
 
 
-class FUVALeftDarkMonitor(BaseMonitor):
-    name = 'FUVA Dark Monitor - Left'
-    data_model = FUVDarkDataModel
+class FUVDarkMonitor(BaseMonitor):
+    """Abstracted FUV Dark Monitor. Not meant to be used directly but rather inherited by specific segment and region
+    dark monitors"""
     labels = ['ROOTNAME']
-    output = # path
-
-    location = (1060, 1260, 296, 734)
+    output = COS_MONITORING
+    docs = "https://spacetelescope.github.io/cosmo/monitors.html#fuv-dark-rate-monitors"
```

Comment:
Just remember to name the section like that when you write the documentation, otherwise the link won't work 😆